README.md

```diff
@@ -56,7 +56,7 @@
 ## Developing
 
 If you are new to React Native, this is a helpful introduction:
-https://facebook.github.io/react-native/docs/getting-started.html
+https://reactnative.dev/docs/getting-started
```

Comment:
Please make the letters rainbow colored for clarity. Besides that, LGTM

Reply:
@markdalgleish either add an emoji here or no approval from me

Reply:
gm

Reply:
gm

Reply:
gm